lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -53,17 +53,21 @@ def inject_embedded_app_options_to_application
       end
 
       def inject_into_application_controller
+        injection = "  include ShopifyApp::Controller\n"
+        if embedded_app?
+          injection << "  layout \"embedded_app\"\n"
+        end
+
         inject_into_class(
           "app/controllers/application_controller.rb",
           'ApplicationController',
-          "  include ShopifyApp::Controller\n"
+          injection
         )
       end
```

Comment:
Since we're bumping major version, could we imagine the app not having to inject anything in an already existing file, but rather simply inject a module with `ActiveSupport.on_load(:action_controller)` in an initializer or something like that? I think @gmalette also had some ideas around that stuff.